wikitemplate-IPFS.md

```diff
@@ -197,13 +197,13 @@
 
 ## Peers
 
-- [ ] Prerequisites: local node launched and local gateway configured on two machines, locally networked (LAN, can be over Wi-Fi).
+- [ ] Prerequisites: Brave local node launched and local gateway configured on two machines, locally networked (LAN, can be over Wi-Fi).
 
    ### `Adding`; see [issue 15567](https://github.com/brave/brave-browser/issues/15567#issuecomment-867983572) for full setup steps
 
    - [ ] Verify when you go to `brave://settings/ipfs/peers` and click on the `Add` button, it prompts you to enter a new peer-connection string. Confirm that entering an incorrect string yields `This name is not valid` upon clicking `Submit`. (Acceptable ones are only CIDs or something like `**/p2p/**` format.)
    - [ ] Verify if a peer is added and node is started, it proposes to restart node to apply changes.
-   - [ ] Verify the local node is restarted by clicking `Restart` button; happen it shows error message and suggests to see more on diagnostic page.
+   - [ ] Verify the Brave local node is restarted by clicking `Restart` button; happen it shows error message and suggests to see more on diagnostic page.
```

Comment:
Should this one also be `Brave local IPFS node` rather than just `Brave local node`?

Reply:
Thanks for flagging; I added `Brave` for clarity but felt `IPFS` was redundant in this context, since I wasn't referring to the button's label/pref.

But given I added `Brave` I should probably go whole-hog and just make them all explicit.